internal/cli/workspace_create.go

```diff
@@ -0,0 +1,96 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/internal/server/gen"
+	"github.com/posener/complete"
+)
+
+type WorkspaceCreateCommand struct {
+	*baseCommand
+
+	flagWorkspaceName string
+}
+
+func (c *WorkspaceCreateCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	flagSet := c.Flags()
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(flagSet),
+		WithNoConfig(),
+		WithNoLocalServer(),
```

Comment:
I can imagine workspaces could be useful to users who only use the local server. Will it not work with a local server?

Reply:
Thanks for bringing this up! I believe I misunderstood what the local server was doing (or supposed to do/offer). After more reading and some discussion, I think this is a mistake. I removed it in 98f0fe07b